### PR TITLE
merge env statements when concatenating

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
@soltysh you might as well look at this one too :)

File: pkg/build/builder/docker.go [hunk 1]
```diff
@@ -188,9 +188,16 @@ func (d *DockerBuilder) addBuildParameters(dir string) error {
 	}
 
 	envVars := getBuildEnvVars(d.build)
+	first := true
 	for k, v := range envVars {
-		newFileData = newFileData + fmt.Sprintf("ENV %s %s\n", k, v)
+		if first {
```
Comment (soltysh) [inline pkg/build/builder/docker.go]: The same problem as with previous, use `k`, don't introduce new vars.

Comment (bparees) [inline pkg/build/builder/docker.go]: k isn't an index here, it's a string key.

Comment (soltysh) [inline pkg/build/builder/docker.go]: Right, I didn't notice envVars is a map here, sorry.

**Comment by soltysh** [comment]
LGTM [merge]

**Comment by bparees** [comment]
[merge]

**Pull request merged**